website/client/components/header/menu.vue

```diff
@@ -18,37 +18,55 @@ div
     b-collapse#menu_collapse.collapse.navbar-collapse
       b-navbar-nav.menu-list
         b-nav-item.topbar-item(tag="li", :to="{name: 'tasks'}", exact) {{ $t('tasks') }}
+          .chevron-block
         li.topbar-item(:class="{'active': $route.path.startsWith('/inventory')}")
+          .chevron-block
+            .chevron-icon-up(v-html="icons.chevronUp")
+            .chevron-icon-down(v-html="icons.chevronDown")
           router-link.nav-link(:to="{name: 'items'}") {{ $t('inventory') }}
           .topbar-dropdown
             router-link.topbar-dropdown-item.dropdown-item(:to="{name: 'items'}", exact) {{ $t('items') }}
             router-link.topbar-dropdown-item.dropdown-item(:to="{name: 'equipment'}") {{ $t('equipment') }}
             router-link.topbar-dropdown-item.dropdown-item(:to="{name: 'stable'}") {{ $t('stable') }}
         li.topbar-item(:class="{'active': $route.path.startsWith('/shop')}")
+          .chevron-block
+            .chevron-icon-up(v-html="icons.chevronUp")
```

Comment:
could you add a `v-once` to these elements (like icons) that never change?

Reply:
Thanks for the tip :]
I correctly understood that I need to add a `v-once` _only_ to the icons?
Like this:
`.chevron-icon-up(v-html="icons.chevronUp", v-once)`

Reply:
So v-once tells Vue not to render the element again which helps with performances. In that case icons seemed the only static thing but in general you can/should use it for every element that's not re-rendere again. For example

```
span {{ $t('someString') }}
```
never changes so it's fine to use v-once on it